scripts/push-dockerimage.sh

```diff
@@ -45,6 +45,9 @@ for tag in ${tags[*]}; do
     docker tag buildtemp "mozilla/balrog:${tag}"
     echo "Pushing Docker image tagged with ${tag}"
     docker push mozilla/balrog:${tag}
+    # Pull the image to print its digest. cloudops-deploylib verifies the
+    # images comparing their digests to the digests in the logs
```

Comment:
Can you revise this comment? I think the `pull` is necessary because the digest changes when you push it (?), but this comment doesn't make it clear.

Reply:
Does pulling defeat the purpose of printing the digest in the log? My assumption is that it is printed so we *don't* have to trust docker.

Reply:
> Does pulling defeat the purpose of printing the digest in the log? My assumption is that it is printed so we _don't_ have to trust docker.

It does. So far I haven't found any better way to find the digest using this old image format... :/

Reply:
After some investigation it turns out that the digest is something registry specific, so you have to pull or push the image to get the digest. Since we push in the same task, there is no need to pull again, `docker push` prints the digest.